images/macos/macos-12-Readme.md

```diff
@@ -237,7 +237,7 @@ mv "/Applications/Visual Studio 2019.app" "/Applications/Visual Studio.app"
 ### Xcode
 | Version          | Build    | Path                           |
 | ---------------- | -------- | ------------------------------ |
-| 14.0 (beta)      | 14A5294g | /Applications/Xcode_14.0.app   |
```

Comment:
Could this version be included in the image still please?

https://developer.apple.com/download/applications/
> For development with the macOS Ventura SDK, continue to use Xcode 14 beta 6.

Reply:
Yeah, I've already thought about it. We will deploy the new image with both Xcodes on Monday-Tuesday.